test/types/fastify-opentelemetry.test-d.ts

```diff
@@ -0,0 +1,50 @@
+import { OpenTelemetryPluginOptions, OpenTelemetryReqInstance } from "../../fastify-opentelemetry";
+import fastify, { FastifyRequest } from "fastify";
+import { expectType } from 'tsd'
+import { Context, Span, Tracer } from "@opentelemetry/api";
+
+// should be able to use openTelemetry functions on FastifyRequest
+fastify().get('/', (req, res) => {
+  expectType<OpenTelemetryReqInstance>(req.openTelemetry());
+  expectType<Span | undefined>(req.openTelemetry().activeSpan);
+  expectType<Context>(req.openTelemetry().context);
+  expectType<Tracer>(req.openTelemetry().tracer);
```

Comment:
Could we also add expects for `inject` and `extract` return values?
```ts
expectType<void>(req.openTelemetry().inject({}))
expectType<Context>(req.openTelemetry().extract({}))
```

Reply:
Added...   thank you for taking the time to review these, it's much appreciated!